### DISCUSSION TITLE
can you set permanent settings per preset?

### DISCUSSION THREAD

**Discussion by arosl** [gopro/labs#1142] [Q&A]
Hi i have not been able to do this so i just want to ask if its possible, and add my use-case of why it could be useful.

Is it possible to have different fourCC codes per preset that is stored in camera and when switching preset it remembers those settings?

My usecase is that my team use multiple GoPro12s to document different things underwater. That often includes using timelapse (photo) mode to make photogrammetry models, one video preset for sitevideos where capturing the maximum amount of data is important and a mode for the occasional photo. 

I can see all the presets when changing mode with the button on the side of the camera, but it does not contain the actual presets before i scan the qrcode again. This being underwater  printing qrcodes before the dive and finding them when switching modes is quite a bit more hassle than just selecting the mode. So if there is a way i can make sure i can lock the ISO between 100-400, no slower shutter than 125, HIST=10 etc in timelapse mode and a small historgram in the corner with the level in video mode without having to scan qrcodes inbetween but just having it being selected by the preset that would be great.

Is this possible or will we have to stick to only camera-wide settings that works in all presets for permanent, and use temporary settings on flickly  qrcodes for switching between the presets?

**Comment by Squinkz** [comment]
This is a feature on the Hero 13's version of Labs afaik.

I too would love this to be ported to the Hero 12!

**Reply by dnewman-gpsw** [reply]
It is coming to HERO12.

**Reply by Squinkz** [reply]
Honestly the best news I've heard all week! Can't wait! Thanks.
